api/src/main/java/org/eclipse/microprofile/faulttolerance/Retry.java

```diff
@@ -30,9 +30,18 @@
 import javax.interceptor.InterceptorBinding;
 
 /**
- * The Retry annotation to define the number of the retries and the fallback method on reaching the
- * retry counts. Any invalid config value causes 
+ * The Retry annotation to define the number of the retries. Any invalid config value causes
  * {@link org.eclipse.microprofile.faulttolerance.exceptions.FaultToleranceDefinitionException}.
+ * <p>
+ * When a method returns and the retry policy is present, the following rules are applied:
+ * <ol>
+ * <li>If the method returns normally (doesn't throw), the result is simply returned.
+ * <li>Otherwise, if the thrown object is assignable to any value in the {@link #abortOn()} parameter, the thrown object is rethrown.
+ * <li>Otherwise, if the thrown object is assignable to any value in the {@link #retryOn()} parameter, the method call is retried
+ *     (up to configured {@link #maxRetries() number of times}).
```

Comment:
I suggest to delete the above line as it does not add much info. It could be up to configured maxRetries or maxDuration.

Reply:
Good point.

Reply:
Done.

Reply:
Should we extend this list to also cover `maxRetries` and `maxDuration`? Possibly in another PR.